azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/UIManager.kt

```diff
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This class wasn't providing any hard value (all it does is launch an intent, which is a standard android action, doesn't need a helper class and function.